src/components/SiteEventsModal.vue

```diff
@@ -42,20 +72,72 @@ export default {
                     sortable: true,
                 },
                 {
-                    field: 'date',
-                    label: 'date',
+                    field: 'UTC',
+                    label: 'UTC',
                     searchable: false,
                     sortable: true,
                 },
+                {
+                    field: 'user',
+                    label: 'user',
+                    visible: this.time_display_format == 'user',
+                    sortable: true,
+                },
+                {
+                    field: 'observatory',
+                    label: 'observatory',
+                    visible: this.time_display_format == 'observatory',
+                    sortable: true,
+                },
                 {
                     field: 'unix',
                     label: 'unix',
                     visible: false,
                     sortable: true,
                 },
-            ],
+            ]
+        },
+    },
+    data() {
+        return {
+            //site_events: [{} * 26],
```

Comment:
Any reason to keep the commented lines in?

Reply:
Nope, good catch.